packages/messaging-api-telegram/src/TelegramClient.ts

```diff
@@ -6,6 +6,7 @@ import omit from 'lodash.omit';
 import snakeCaseKeys from 'snakecase-keys';
 import urlJoin from 'url-join';
 import { onRequest } from 'messaging-api-common';
+import { pick } from 'lodash';
```

Comment:
Should we change this to `import pick from 'lodash/pick';`?

cc @chentsulin

Reply:
I would vote for 

```js
import pick from 'lodash/pick';
```

Reply:
Fixed.